run_instructions.txt

```diff
@@ -0,0 +1,4 @@
+python training.py perfect-static resnet18 --pretrained --gpu 2 --local_data data/PerfectStaticTextures/
```

Comment:
Please put `#!/usr/bin/env bash` at the top of the file.

And `run_instructions` does not need a `.txt` extension. Either go with `.sh` or no extension.